apps/etterlatte-beregning/src/main/kotlin/BeregningRepository.kt

```diff
@@ -134,7 +134,8 @@ private fun toBeregning(beregningsperioder: List<BeregningsperiodeDAO>): Beregni
                 utbetaltBeloep = it.utbetaltBeloep,
                 soeskenFlokk = it.soeskenFlokk,
                 grunnbelopMnd = it.grunnbelopMnd,
-                grunnbelop = it.grunnbelop
+                grunnbelop = it.grunnbelop,
+                trygdetid = 40 // TODO: Må fikses før vi tar imot saker som IKKE har 40 års trygdetid
```

Comment:
Finnes det en jira på dette? Hvis vi legger inn tickeren så er det enklere finne alle disse stedene når man tar oppgaven 🚀

Reply:
Ja, og nja... Har koblet den på EY-1403 som omhandler beregningsperioder i brev, men kan slenge på kobling mot [EY-981](https://jira.adeo.no/browse/EY-981) som omhandler trygdetid i beregning. Bare for å ha en ekstra kobling.

Reply:
Do it!